configs/eslint/package.json

```diff
@@ -11,14 +11,14 @@
   "publishConfig": {
     "access": "public"
   },
-  "devDependencies": {
-    "@eslint/compat": "^1.2.0",
-    "@eslint/js": "^9.12.0",
-    "eslint": "^9.16.0",
-    "eslint-plugin-react": "^7.37.1",
+  "dependencies": {
```

Comment:
Just making sure we intended to move a lot of these from dev deps into deps. If so, that's fine.